content/en/tracing/trace_explorer/trace_view.md

```diff
@@ -21,23 +21,85 @@ further_reading:
+## Trace visualizations
 
-{{< img src="tracing/visualization/trace/trace_list.png" alt="Trace list" style="width:90%;">}}
+{{< tabs >}}
+{{% tab "Flame Graph" %}}
+
+{{< img src="tracing/trace_view/flamegraph.png" alt="Flame Graph" style="width:90%;">}}
+
+The default visualization that displays all the spans for an associated trace, color-coded on a timeline. This is useful for understanding the execution path of a request and where time was spent over a trace.
+
+To navigate the graph, scroll to zoom, click and drag to move around, and use the minimap to zoom into the selected span or zoom out to the full trace.
+
+The legend details the color coding of the flame graph. Group spans by either **Service** (default), **Host**, or **Container**. Choose to display either the percentage of trace execution time (**% Exec Time**) or span count (**Spans**) by color grouping. If errors exist on spans in the trace, highlight them in the flame graph by selecting the **Errors** checkbox under **Filter Spans**.
+
+{{< img src="tracing/trace_view/flamegraph_legend.mp4" alt="Flame Graph legend" video="true" style="width:90%;">}}
+
+
+{{% /tab %}}
+{{% tab "Span List" %}}
+
+{{< img src="tracing/trace_view/spanlist.png" alt="Trace View" style="width:90%;">}}
+
+Displays [resources][1] by grouping ([service][2] by default) and sorts them according to their count of spans. This visualization is useful for scanning latency information by resource or grouping.
+
+Filter resources by type or naming information using the corresponding buttons and text-based search.
+
+{{< img src="tracing/trace_view/spanlist_headers.png" alt="Span List headers" style="width:90%;">}}
+
+Sort groupings by clicking on the corresponding column header: **RESOURCE**, **SPANS**, average duration (**AVG DURATION**), execution time (**EXEC TIME**), or percentage of trace execution time (**% EXEC TIME**).
+
+[1]: /tracing/glossary/#resources
+[2]: /tracing/glossary/#services
+{{% /tab %}}
+{{% tab "Waterfall (Beta)" %}}
+
+<div class="alert alert-info"><strong>Join the Beta!</strong><br />
+To join the Waterfall private beta <a href="https://forms.gle/LjJR1ZbF1tNDv5JC6">fill out this form</a>.</div>
+
+{{< img src="tracing/trace_view/waterfall.png" alt="Waterfall" style="width:90%;">}}
+
+Displays all the spans for an associated trace, color-coded on separate rows and on a timeline. This visualization is useful for isolating relevant parts of a trace and understanding how asynchronous processes fit into the execution flow.
+
+On each row (that is, per span):
+* A bar (colored by service), whose length corresponds to the percentage of total trace duration
+* The service name, operation name, and resource name, with font styling: **service** operation <span style="color:gray">resource</span>
```

Comment:
This sentence is not clear to me

Reply:
It refers to the visual part of the visualization, where each span has a bar:
![image](https://github.com/DataDog/documentation/assets/88278697/0595bda1-5abc-4dcd-9564-ed5b35fe42be)
Any recommendations for how to word this differently? It's tough to describe and easy to see/understand

Reply:
The first bullet point on the bar is fine to me. it is more regarding the service name/operation/resource.
What about annotating on a screenshot?